css/styles.css

```diff
@@ -0,0 +1,64 @@
+body {
+  max-width: 320px;
+  margin: 0 auto;
+}
```

Comment:
[OPTIONAL] Well done for implementing the changes requested by the previous reviewer. :1st_place_medal: However, I can see that the code on line 3 `max-width: 320px` is the cause of the problem. :stuck_out_tongue_winking_eye: Limiting the width of the whole body to 320px means that your page will not be responsive anymore and the width of the whole body cannot exceed 320px. :wink: Kindly consider removing that line to make your page more responsive. :pray: 

`NOTE`: _I would normally have requested you to make the changes before approval but I am making it optional since the previous reviewer specifically told you what to do which you have done. However, I strongly suggest you work on it since it may affect your the dimensions of your website_

Reply:
@jerryowusu Hi Jerry, the Figma template provided a mobile version width of 320px, so i thought that should be the width of the page. I misinterpreted that part. I will remove that max-width property. Thank you